commit c10d840ff1f138dd5d8bdfca7d18673f0e30de05
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Sun May 31 10:14:17 2015 +0200

    python: Make it optional
    
    python bindings are built unless disabled

diff --git a/configure.ac b/configure.ac
index cdee862..c15d825 100644
--- a/configure.ac
+++ b/configure.ac
@@ -15,7 +15,6 @@ AC_PREREQ([2.63])
 AC_INIT([libixion], [ixion_version])
 AC_CONFIG_HEADERS([config.h])
 AM_INIT_AUTOMAKE([1.11 foreign dist-bzip2 dist-xz])
-AM_PATH_PYTHON(3)
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 AX_CXX_COMPILE_STDCXX_11
 
@@ -29,6 +28,9 @@ AC_ARG_WITH(mdds-include-path,
     AS_HELP_STRING([--with-mdds-include-path], [specify include path for mdds headers.])
 ,)
 
+AC_ARG_ENABLE([python],
+    AS_HELP_STRING([--disable-python], [Disable python bindings]))
+
 IXION_API_VERSION=ixion_api_version
 IXION_MAJOR_VERSION=ixion_major_version
 IXION_MINOR_VERSION=ixion_minor_version
@@ -83,7 +85,11 @@ fi
 PKG_CHECK_MODULES([MDDS],[mdds >= 0.12.0])
 
 # Check for python.
-PKG_CHECK_MODULES([PYTHON], [python3 >= 0.28])
+AS_IF([test "x$enable_python" != "xno"], [
+    AM_PATH_PYTHON(3)
+    PKG_CHECK_MODULES([PYTHON], [python3 >= 0.28])
+])
+AM_CONDITIONAL([PYTHON], [test "x$enable_python" != "xno"])
 
 CPPFLAGS="$CPPFLAGS -g -Os -fvisibility=hidden"
 
diff --git a/src/Makefile.am b/src/Makefile.am
index 6e138a4..cd912ab 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -1,4 +1,7 @@
-SUBDIRS = libixion python
+SUBDIRS = libixion
+if PYTHON
+	SUBDIRS += python
+endif
 
 AM_CPPFLAGS = -I$(top_srcdir)/include $(MDDS_CFLAGS)
 
